tests/unit/renderers/editor-dom-test.js

```diff
@@ -172,6 +172,27 @@ test('renders a post with image', (assert) => {
   assert.equal(renderTree.rootElement.innerHTML, `<img src="${url}">`);
 });
 
+test('renders a post with atom', (assert) => {
+  let post = builder.createPost();
+  let section = builder.createMarkupSection('P');
+  post.sections.append(section);
+
+  section.markers.append(
+    builder.createAtom('mention', '@bob', {})
+  );
```

Comment:
This can be done a bit more simply with the test helper for postAbstract:
```
let post = Helpers.postAbstract.build(builder => {
  let { markupSection, post, atom } = builder;
  return post([markupSection('p', [atom('mention', '@bob', {})])]);
});
```